codegen/src/main/kotlin/org/jetbrains/bsp/generators/CodegenFile.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.bsp.generators
```

Comment:
What's the point of translating `codegen` to Kotlin? Why do we need to change the package name?

Reply:
Our new generators (for Kotlin and Rust) are written in Kotlin and use the `org.jetbrains.bsp` package name. I didn't include them here, but you can see them [here](https://github.com/agluszak/bsp-generators)